### DISCUSSION TITLE
Serius problem with desktop Version

### DISCUSSION THREAD

**Discussion by jimhere** [Shabinder/SpotiFlyer#180] [General]
I recently downloaded the desktop version. When i download a playlist or a song , I cant find where the songs where downloaded. Even when i hit the settings button and follow the path, i still cant find anything. Do i do something wrong ?

**Comment by Shabinder** [comment]
using windows?

**Reply by jimhere** [reply]
Hello Shabinder and thank you for replying so quickly, 

At the beggining yes i couldnt find the folder in the path that i selected . But now i can easily find them. Sometimes appears them in desktop , other times just in users/administrator and definately not in documents. I must say that the app is great and has room for improvements.  Donation is on the way. 

You are great , keep making us happy !
RadioLab team <3

**Reply by Shabinder** [reply]
Hey @jimhere,
I agree there is a lot of scope for improvements and I am working on the same...
and Thanks for taking the initiative to **support** me :+1:, keeps me encouraged :smiley:

**Comment by densma1** [comment]
no luck here. do you have to login to download? cant find file so not even sure if download works on not. using windown 10
